src/applications/fry-dea/config/form.js

```diff
@@ -539,7 +542,7 @@ const formConfig = {
               'ui:title': (
                 <>
                   <span className="fry-dea-labels_label--main vads-u-padding-left--1">
-                    Which education benefit would you like to apply for?
+                    <b>Which education benefit would you like to apply for?</b>
```

Comment:
First of all, `<b>` tags are deprecated in favor of `<strong>` (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b). However, I don't think this label should be in boldface. That is not the default for the Forms System (see other pages). I do see some of the text in one of the mockups in boldface, but I think that's a mistake. It should only be bold when that form group is in an error state.

Reply:
Good to know about the <b> tags.  QA wanted the change since that's what the design had, i'll run it by them.